bin/upload_cases_sheet.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+set -e
+
+aws s3 cp --sse aws:kms --sse-kms-key-id $AWS_KMS_KEY_ID ../data/$1.xlsx s3://$AWS_BUCKET/uans/$1
```

Comment:
Just wondering, if this form uses uans, ceps and brps should you not have a more generic folder name than uans?

Reply:
the project will be using uan/brp and dob not confirmed yet but we will implement something similar to ima